setup.py

```diff
@@ -208,9 +208,9 @@ def build_extensions(self):
     "Operating System :: Microsoft :: Windows",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3 :: Only",
```

Comment:
@trbromley would it be good to reflect this already in the v0.17.0 release? Will v0.17.0 discontinue support for Python 3.6/add support for Python 3.9?

Reply:
@antalszava yes this would be good to merge into the release branch too - I think we should drop 3.6 support in this release since it's not anyway currently supported in current PL.

Reply:
Sounds great! @albi3ro could you maybe delete the `    "Programming Language :: Python :: 3.6",` line from `setup.py` in #127?